SPARCore/GADArticle.m

```diff
@@ -12,35 +11,74 @@ @implementation GADArticle
 static NSString *const API_PUBLICATION_ID = @"publication";
 static NSString *const API_TITLE = @"title";
 
-static NSString *const API_HOSTNAME = @"https://g2j7qs2xs7.execute-api.us-west-2.amazonaws.com/";
-static NSString *const API_PREFIX = @"devstable";
+static NSString *const API_ITMES = @"items";
+static NSString *const API_NEXT_PAGE_TOKEN = @"nextPageToken";
+
 static NSString *const API_PUBLICATION_PATH = @"publications";
 static NSString *const API_ARTICLE_PATH = @"articles";
 
-+ (void) articlesFromPublication: (NSString *)publicationId
-                                        completionHandler:(void(^_Nonnull)(NSArray<GADArticle *>
-                                                                           *_Nullable articles,
-                                                            NSError *_Nullable error))completion {
-    NSURL *queryURL = [GADArticle urlForArticlesFromPublication:publicationId];
++ (void) articlesForPublicationId: (NSString *)publicationId
```

Comment:
I've thought about this,too. But then I'll have to move those constant string of json key of article to the publication class, which doesn't seem very right...